tests/find/test-suite-1/test.escaping.js

```diff
@@ -0,0 +1,134 @@
+'use strict';
+
+testCases.push(function (dbType, context) {
+
+  describe.skip(dbType + ': test.escaping.js', function () {
```

Comment:
Will this be something we can support later?

Reply:
Yeah, it's skipping currently because if it runs it crashes the whole rest of the test stack. When we merge tests early we'll re-enable this